workflows/cloudify_system_workflows/snapshots/utils.py

```diff
@@ -254,15 +254,10 @@ def db_schema_downgrade(revision='-1'):
     :type revision: str
 
     """
-    subprocess.check_call([
```

Comment:
factored out the common logic (the subprocess call, passing the --postgresql-host parameter) to a separate function